src/VisualStudio/Core/Test/CodeModel/MethodXML/MethodXMLTests_CSInvocations.vb

```diff
@@ -255,9 +255,14 @@ public class C
                 <Type>System.String</Type>
                 <Argument>
                     <Expression>
-                        <Literal>
-                            <Char>.</Char>
-                        </Literal>
```

Comment:
Yes, but only slightly. Remember, that this is never seen by the user and the designer treats it the same way. I felt it better to err on the side of "only embed letters and digits" rather than try to sort out precisely which characters are legal in XML.